test/jest/index.js

```diff
@@ -0,0 +1,4 @@
+import { createSwapsMockStore } from './mock-store';
+import { renderWithProvider } from './rendering';
+
+export { createSwapsMockStore, renderWithProvider };
```

Comment:
FYI, the reason for this file is to export all shared functions from `test/jest` and then when we need to import some of them in our tests, we only need to import this file instead of multiple files.